pkg/dev/envvars_test.go

```diff
@@ -88,6 +90,118 @@ func TestSetEnvVarsOverride(t *testing.T) {
 	require.True(requireEnvVarExists(envVars, "ENV_VAR_FROM_VALUE", envVarValueFromDevConfig))
 }
 
+func TestGetEnvVarsForView(t *testing.T) {
+	testCases := []struct {
+		desc    string
+		config  GetEnvVarsForViewConfig
+		envVars map[string]string
+	}{
+		{
+			desc: "Gets env vars from value from view",
+			config: GetEnvVarsForViewConfig{
+				ViewEnvVars: map[string]libapi.EnvVarValue{
+					"ENV_VAR_FROM_VALUE": {
+						Value: pointers.String("foo"),
+					},
+				},
+			},
+			envVars: map[string]string{
+				"ENV_VAR_FROM_VALUE": "foo",
+			},
+		},
+		{
+			desc: "Gets env vars from config from dev config file",
+			config: GetEnvVarsForViewConfig{
+				ViewEnvVars: map[string]libapi.EnvVarValue{
+					"ENV_VAR_FROM_CONFIG": {
+						Config: pointers.String("my_config"),
+					},
+				},
+				ConfigVars: map[string]env.ConfigWithEnv{
+					"my_config": {
+						Config: api.Config{
+							Name:  "my_config",
+							Value: "foo",
+						},
+					},
+				},
+			},
+			envVars: map[string]string{
+				"ENV_VAR_FROM_CONFIG": "foo",
+			},
+		},
+		{
+			desc: "Gets env vars from value from dev config file",
+			config: GetEnvVarsForViewConfig{
+				DevConfigEnvVars: map[string]string{
+					"ENV_VAR_FROM_VALUE": "foo",
+				},
+			},
+			envVars: map[string]string{
+				"ENV_VAR_FROM_VALUE": "foo",
+			},
+		},
+		{
+			desc: "Env var in dev config file same env var in view",
+			config: GetEnvVarsForViewConfig{
+				DevConfigEnvVars: map[string]string{
+					"ENV_VAR_FROM_VALUE": "foo",
```

Comment:
```suggestion
					"ENV_VAR_FROM_VALUE": "bar",
```

This way, you can test that the dev config env vars override the view's env vars. Right now, this test isn't too meaningful